package.json

```diff
@@ -11,7 +11,9 @@
     "lint:check-frontmatter": "node ./scripts/check-frontmatter.mjs",
     "lint:validation": "npm run build",
     "build": "tsx ./scripts/build-docs.ts",
-    "test": "vitest --silent"
+    "test": "vitest --silent",
+    "typedoc:link": "node scripts/typedoc-link.mjs",
```

Comment:
We'll need to document this

Reply:
Yes, I explained in this step here: https://github.com/clerk/clerk#5-set-up-local-docs-support
So I think for actually working on the docs the `typedoc:link` script isn't even necessary